packages/calypso-e2e/src/lib/blocks/block-flows/layout-grid.ts

```diff
@@ -73,23 +76,26 @@ export class LayoutGridBlockFlow implements BlockFlow {
 		columnDetails: ColumnDetails,
 		context: EditorContext
 	): Promise< void > {
-		const popoverBlockInserter = new PopoverBlockInserterComponent(
-			context.editorIframe,
-			context.page
-		);
+		const openInlineInserter: OpenInlineInserterDelegate = async ( editor: Locator ) => {
+			const addBlockButtonLocator = editor.locator(
+				selectors.addBlockButton( columnDetails.columnNumber )
+			);
+			// On mobile, a lot of clicks are eaten en route to the button. This doesn't play well with Playwright and can cause fragility.
```

Comment:
I assume the 'button' here refers to the 'add block' button?

Reply:
Yep! 👍 

Let me know if you think that could use some clarification in the code!